OpenSim/Simulation/Model/Geometry.h

```diff
@@ -155,12 +144,12 @@ class OSIMSIMULATION_API Geometry : public Component {
     };
 
     /// Convenient access to set Appearance/representation
-    void setRepresentation(const DisplayPreference& rep) { 
+    void setRepresentation(VisualRepresentation rep) { 
         upd_Appearance().set_representation(rep); 
     };
     /// Convenient access to get Appearance/representation
-    const DisplayPreference& getRepresentation() { return 
-        (const DisplayPreference&)get_Appearance().get_representation(); 
+    VisualRepresentation getRepresentation() { return
+        get_Appearance().get_representation(); 
```

Comment:
Having this list of conveniences that set directly on the members of Appearance further negates reasons to separate surface and curve appearances, in my view.

I vote against these convenience accessors in favor of getting and setting an Appearance. That makes handling the notion of a shared Appearance more straightforward, I think.